packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -182,11 +182,7 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 									isPrimary
 									disabled={ !! disabledLabel }
 								>
-									<span>
-										{ disabledLabel
-											? __( disabledLabel, __i18n_text_domain__ )
-											: __( 'Choose', __i18n_text_domain__ ) }
-									</span>
+									<span>{ disabledLabel ?? __( 'Choose', __i18n_text_domain__ ) }</span>
```

Comment:
No need to wrap `disabledLabel` in a `__()` here, since it's already a translated string — see [here](https://github.com/Automattic/wp-calypso/blob/7e2b4def44afda7529452950005e9b820995e079/apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx#L73) and [here](https://github.com/Automattic/wp-calypso/blob/f140949f83baca52118fe47b678661b9caf6fde0/packages/launch/src/focused-launch/plan-details/index.tsx#L76)